R/pjn_metadata.R

```diff
@@ -46,3 +49,17 @@ get_pjn_region <- function(pjn) {
     return(region)
   }
 }
+
+#' Get subnational region code from parsed PJN
+#'
+#' @keywords internal
+get_pjn_region_code <- function(pjn) {
+  idx <- which(pjn[, "Tag"] == "<Projection Parameters - Subnational Region Name2>") + 3
+  region_code <- pjn[idx, "Data"]
```

Comment:
This looks like a more readable way of doing it than I do above. Do you want to update that one too?